wagtail/admin/views/pages/edit.py

```diff
@@ -32,6 +34,81 @@
 )
 
 
+class BaseSidePanel(Component):
```

Comment:
Eventually I think we will need all of those in CreateView as well, so should we move those definitions to a separate file already?

Reply:
Done, thanks for pointing this out!